internal/controller/utils/nim.go

```diff
@@ -484,6 +484,10 @@ func GetNimServingRuntimeTemplate(scheme *runtime.Scheme) (*v1alpha1.ServingRunt
 								MountPath: "/opt/nim/workspace",
 								Name:      "nim-workspace",
 							},
+							{
+								MountPath: "/.cache",
+								Name:      "nim-workspace",
```

Comment:
did you test it?
I'm not sure if you can bind multiple mountPaths to a single pv, I was referring to using this:
```yaml
{
		MountPath: "/mnt/models/cache",
		Name:      "nim-pvc",
},
```

And what is not clear to me, why do you need two caches?

Reply:
@TomerFi updated to separate volume for /.cache